ephypype/fif2ts.py

```diff
@@ -19,3 +19,66 @@ def ep2ts(fif_file):
     save_path = op.abspath('ts_epochs.npy')
     save(save_path, data)
     return save_path
+
+
+def create_ts(raw_fname):
+    """Read a raw data from **.fif** file and save the data time series, the
+    sensors coordinates and labels in .npy and .txt files.
+
+    Parameters
+    ----------
+    raw_fname : str
+        pathname of the raw data to read
+
+    Returns
+    -------
+    ts_file : str
+        pathname of the numpy file (.npy) containing the data read from
+        raw_fname
+    channel_coords_file : str
+        pathname of .txt file containing the channels coordinates
+    channel_names_file : str
+        pathname of .txt file containing the channels labels
+    sfreq : float
+        sampling frequency
+    """
+    import os
+    import numpy as np
+
+    import mne
+    from mne.io import read_raw_fif
```

Comment:
why not import these globally? Otherwise you will do an import for every function call, thus making the functions slower ...